tests/components/bond/test_config_flow.py

```diff
@@ -0,0 +1,94 @@
+"""Test the Bond config flow."""
+from requests.exceptions import ConnectionError
+from simplejson import JSONDecodeError
+
+from homeassistant import config_entries, setup
+from homeassistant.components.bond.const import DOMAIN
+from homeassistant.const import CONF_ACCESS_TOKEN, CONF_HOST
+
+from tests.async_mock import patch
+
+
+async def test_form(hass):
+    """Test we get the form."""
+    await setup.async_setup_component(hass, "persistent_notification", {})
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+    assert result["type"] == "form"
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5802827)